### ISSUE TITLE
Make RenderTargetBitmap support CopyPixels

### ISSUE THREAD

**Issue by jgcodes2020** [AvaloniaUI/Avalonia#12329]
**Is your feature request related to a problem? Please describe.**

See this discussion: https://github.com/AvaloniaUI/Avalonia/discussions/10280

I've run into the same exact problem. The only semi-legitimate method of getting what I want is to use reflection (see above discussion), as serializing/deserializing an image is REALLY slow.

**Describe the solution you'd like**
As OP suggested in the discussion above, I'd like to see `RenderTargetBitmap.CopyPixels` implemented, i.e. have `Avalonia.Skia.SurfaceRenderTarget` implement `IReadableBitmapImpl`.

**Describe alternatives you've considered**
TL;DR:
- Save the image to a `MemoryStream` and then load it again. (Too slow, I need to do this every frame)
- Use reflection (isn't officially supported)
- Call `ResizeBitmap` to copy it to a different bitmap that can be read (not possible)

**Comment by kekekeks** [comment]
Should  be already supported since https://github.com/AvaloniaUI/Avalonia/pull/11235

**Comment by timunie** [comment]
This API is already available, see: https://api-docs.avaloniaui.net/docs/T_Avalonia_Media_Imaging_RenderTargetBitmap#methods
